### DISCUSSION TITLE
Thanks! 🙏

### DISCUSSION THREAD

**Discussion by dfahlander** [dexie/Dexie.js#1788] [General]
Thanks to:

@simon-lammes

@floer32

@secondl1ght

@TomasHubelbauer

...for sponsoring Dexie.js!

🙏

**Comment by dfahlander** [comment]
Thanks also to:

@lambdatest

...for sponsoring Dexie.js on [Open Collective](https://opencollective.com/dexiejs)

And for all historical sponsorships from:

@github

@aecreations

@barbalex

@reeseschultz

🙏

**Comment by secondl1ght** [comment]
Dexie is so great! I love building with it and will my share my app when it is complete. It will be open source so you can look at how I am using Dexie too if you are interested. :)

**Reply by secondl1ght** [reply]
@dfahlander here is my app that I built with Dexie: https://github.com/secondl1ght/cipherchat. I just open sourced it today, thanks again!
